src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,41 @@
+package duke;
+
+import duke.command.Command;
+
+/**
+ * Duke is a Chatbot that is able to store and keep track of tasks inputted by the user.
+ */
+public class Duke {
+
+    private static final String FILE_PATH = System.getProperty("user.dir") + "/data/tasks.txt";
```

Comment:
Is this really necessary if you use relative paths? Not to say it's wrong but maybe could be simpler :)